src/main/resources/native/android/c/attach_adapter.c

```diff
@@ -65,6 +66,17 @@ void registerAttachMethodHandles(JNIEnv* androidEnv) {
     if ((t == NULL) && (jtmp != NULL)) {
         jKeyboardServiceClass= (jclass)(*androidEnv)->NewGlobalRef(androidEnv, jtmp);
     }
+
+    // Position
+    jtmp = (*androidEnv)->FindClass(androidEnv, "com/gluonhq/helloandroid/DalvikPositionService");
```

Comment:
At some point we should refactor this into one method that can be applied to each of the services